website/docs/r/batch_job_definition.html.markdown

```diff
@@ -256,6 +290,146 @@ The following arguments are optional:
 * `secret_name` - The name of the secret. The name must be allowed as a DNS subdomain name.
 * `optional` - (Optional) Specifies whether the secret or the secret's keys must be defined.
 
+### `ecs_properties`
+
+* `task_properties` - The properties of the Amazon ECS task definition on a job. See [`task_properties`](#task_properties) below.
+
+### `task_properties`
+
+* `containers` - The properties of the container. See [containers](#ecs_containers) below.
+* `ephemeral_storage` - (Optional) The ephemeral storage settings for the task. See [ephemeral_storage](#ephemeral_storage) below.
+* `execution_role_arn` - (Optional) The Amazon Resource Name (ARN) of the IAM role that the AWS Batch can assume.
+* `ipc_mode` - (Optional) The IPC resource namespace to use for container in the task. The valid values are `host`, `task`, or `none`.
+* `network_configuration` - (Optional) The network configuration for the Amazon ECS task. See [network_configuration](#network_configuration) below.
+* `pic_mode` - (Optional) The process namespace to use for the container in the task. The valid values are `host` or `task`.
```

Comment:
```suggestion
* `pid_mode` - (Optional) The process namespace to use for the container in the task. The valid values are `host` or `task`.
```

Reply:
Thank you! fixed https://github.com/hashicorp/terraform-provider-aws/pull/37871/commits/6a966d7f6a9f6e3c0bcad71d82e61e4cb22fe9fa